src/modules/fetchData.js

```diff
@@ -7,7 +9,7 @@ const fetchData = async () => {
       let output = '';
       sorted.forEach((el) => {
         output += `
-          <li>name: ${el.user}, score: ${el.score}</li>
+          <li class="p-2 d-flex justify-content-around"> <span class="font-weight-bold">Name:</span> ${el.user} <span class="font-weight-bold"> Score:</span> ${el.score}</li>
           `;
       });
       document.querySelector('#score_listing').innerHTML = output;
```

Comment:
- [OPTIONAL] I would recommend you  to clear the input fields after every submit for good user experience.